### ISSUE TITLE
tools: custom-elements.json is loading from element root instead of project root in RHDS

### ISSUE THREAD

**Issue by zeroedin** [patternfly/patternfly-elements#2010]
## Description of the issue

When using `pfe-tools` in the RHDS repo, the `<api-viewer>` component by way of demo.ts looks for the custom-elements.json file in the root of the element which returns a 404.

```
GET http://localhost:8000/elements/rh-secondary-nav/custom-elements.json 404 (Not Found)
```

demo.ts:149
```
viewer.src = `/${base}/${element}/custom-elements.json`;
```

This `src` location is also used by manifest.ts which also throws an error subsequently.

```
manifest.ts:63 SyntaxError: Unexpected token N in JSON at position 0
```

### Expected behavior

For RHDS repo being a non-monorepo, the `custom-elements.json` should load from the project root.

**Comment by bennypowers** [comment]
it's more than that, we'll also have to set an attribute on the viewer element to select the particular element

**Comment by bennypowers** [comment]
Probably fixed by #2017
